stable/airflow/templates/secrets.yaml

```diff
@@ -13,4 +13,5 @@ data:
   postgresUser: {{ .Values.postgresql.postgresUser | b64enc | quote }}
   postgresPassword: {{ .Values.postgresql.postgresPassword | b64enc | quote }}
   redisPassword: {{ .Values.redis.password | b64enc | quote }}
+  ldapBindPassword: {{ .Values.ldap.bindPassword | b64enc | quote }}
```

Comment:
this is good for demo/test/poc (existingAirflowSecret set to false) but in prod this should not appear